howitz.toml.example

```diff
@@ -15,6 +16,10 @@ server =
 #[zino.connections.other]
 #server =
 
+[caching]
+CACHE_TYPE = 'FileSystemCache'
+CACHE_DIR = './.howitz_cache'
+
```

Comment:
Maybe these should live under `flask`.

Reply:
What I mean is: I suspect we don't need to parse these but that they can be picked from the environment in the same way as flask's own config vars.